glib/src/value.rs

```diff
@@ -297,6 +297,26 @@ mod private {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Given a type implementing `From<i32>`, the type can easily and safely
+/// implement `FromValue` by doing
+/// ```
+/// impl FromValueDelegate for MyInt {
+///   type Delegate = i32;
+/// }
+/// ```
+pub trait FromValueDelegate<'a>: From<Self::Delegate> {
+    type Delegate: FromValue<'a>;
```

Comment:
Wouldn't be nicer to also automatically get the `ToValue` `impl` here at the same time? `ToValue` is safe to implement but if we require `From` in both directions I think having a single `ValueDelegate` trait like this would be nicer.